### ISSUE TITLE
[UI] Add visual "seen" cue to show which markers have been opened and which videos played

### ISSUE THREAD

**Issue by rudokemper** [Terrastories/terrastories#217]
We should add some kind of a visual (like an eye graphic) to show users which markers they've clicked on, and which videos they've seen in a session.

**Comment by mirandawang** [comment]
@rudokemper Is this only for logged in users?

If so, we could keep track of what they've clicked on/seen in the database.

Otherwise we could possibly use cookies which would be based on the browser and device.

**Comment by rudokemper** [comment]
Good question. I'd be curious to know what others think too, and also from a technical point of view in terms of feasibility.

My thinking is that it should be available for anonymous users too, so then based on session cookies. Can those be set to expire? Multiple users might be using the same device at different times, so then the "seen" cues could reset.

**Comment by lanpai** [comment]
Cookies can be set to expire on certain dates or once the session ends (closing the browser). Would many people be sharing computers?

**Comment by rudokemper** [comment]
@lanpai yes, they will be. My thinking is that expiration upon closing the browser is probably the best approach.
